dev-packages/ffmpeg/src/replace-ffmpeg.ts

```diff
@@ -50,7 +50,8 @@ export async function replaceFfmpeg(options: ffmpeg.FfmpegOptions = {}): Promise
     if (shouldDownload) {
         const ffmpegZipPath = await electronGet.downloadArtifact({
             version: electronVersion,
-            artifactName: 'ffmpeg'
+            artifactName: 'ffmpeg',
+            arch: process.env.ARCH
```

Comment:
Even with this line, we need https://github.com/eclipse-theia/theia/pull/10734/files#diff-88ba4f62eeb3a9a874dab680de4ba741825e6657155d6e6d0f5cf5cea7389f60R185-R193 ?

Reply:
I would have assumed `@electron/get` would already detect the arch by default too?

Reply:
This line downloads libffmpeg.dylib to use electron (arm64) runtime with ARCH environment variable.